scripts/Install-VSCode.ps1

```diff
@@ -439,7 +455,7 @@ function Install-VSCodeFromTar {
 
 # We need to be running as elevated on *nix
 if (($IsLinux -or $IsMacOS) -and (id -u) -ne 0) {
```

Comment:
It looks like `-User` editions only apply on Windows (we could install anywhere on Linux from a tar and possibly the same on macOS, but it's likely more complexity than it's worth).

In that case, you might want to add another error message here when the `$BuildEdition` ends in `User` but we're on macOS or Linux, to say something like `VSCode user installation is only supported on Windows`

Reply:
There is a similar check [here](https://github.com/Lothindir/vscode-powershell/blob/f7095bc01b5a2c9b76ed72d0c9a4ffcabe7800a6/scripts/Install-VSCode.ps1#L226).
If we add another error message it'll be doubled if the platform is *nix and the `$BuildEdition` is `*-User`

Reply:
There is a similar check at [line 226](https://github.com/Lothindir/vscode-powershell/blob/f7095bc01b5a2c9b76ed72d0c9a4ffcabe7800a6/scripts/Install-VSCode.ps1#L226).
It would fire after your suggestion.

Reply:
Done in b696783a7eef96e2a28bb41bd3e287b77970141c